docs/book/advanced-guide/pipelines/containerization.md

```diff
@@ -211,6 +231,28 @@ def my_pipeline(...):
     ...
 ```
 
+If you want to use this image directly to run your steps without
+including any code or installing any requirements on top of it,
+you can skip the Docker builds by specifying it in the Docker
+settings:
+
+```python
+docker_settings = DockerSettings(
+    parent_image="my_registry.io/image_name:tag",
+    skip_build=True
+)
+
+@pipeline(settings={"docker": docker_settings})
+def my_pipeline(...):
+    ...
+```
+
+{% hint style="warning" %}
+This is an advanced feature and will most likely break your pipelines. If you use this,
+you're on your own and need to make sure your code files and global configuration
```

Comment:
```suggestion
This is an advanced feature and will most likely cause unintended and unanticipated behavior when running your pipelines. If you use this, make sure your code files and global configuration
```

'You're on your own' felt a bit bleak...

Reply:
You're right, the same warning is in a few other parts of that page to discourage actually doing this. I'll update all of them with your suggestions, seems way better